tools/gyp/pylib/gyp/generator/make.py

```diff
@@ -310,7 +310,7 @@ def CalculateGeneratorInputInfo(params):
 # We write to a dep file on the side first and then rename at the end
 # so we can't end up with a broken dep file.
 depfile = $(depsdir)/$(call replace_spaces,$@).d
-DEPFLAGS = -qmakedep -MF $(depfile).raw
+DEPFLAGS = -qmakedep -qLONGLONG -qLANGLVL=EXTENDED -D_UNIX03_THREADS -D_UNIX03_SOURCE -D_OPEN_SYS_IF_EXT=1 -qCHARS=signed -qXPLINK -qFLOAT=IEEE -D_OPEN_MSGQ_EXT -D_XOPEN_SOURCE_EXTENDED -D_ALL_SOURCE -D_LARGE_TIME_API -DUTMP_FILE=\"/etc/utmpx\" -DPATH_MAX=255 -D_OPEN_SYS_SOCK_IPV6 -D_OPEN_SYS_FILE_EXT -DIP_TTL=-1 -DNI_MAXHOST=1024 -DNI_MAXSERV=32 -MF $(depfile).raw
```

Comment:
This is not from the latest commit. The latest commit (titled "Linux compatibility") will use the appropriate flags for zOS and non-zOS platforms.